src/unix/mod.rs

```diff
@@ -1423,6 +1423,18 @@ cfg_if! {
     }
 }
 
+cfg_if! {
+    if #[cfg(not(target_os = "android"))] {
+        extern "C" {
+            #[cfg_attr(
+                all(target_os = "macos", target_arch = "x86"),
+                link_name = "confstr$UNIX2003"
+            )]
+            pub fn confstr(name: ::c_int, buf: *mut ::c_char, len: ::size_t) -> ::size_t;
+        }
+    }
+}
```

Comment:
```suggestion
#[cfg(not(target_os = "android"))]
extern "C" {
    #[cfg_attr(
        all(target_os = "macos", target_arch = "x86"),
        link_name = "confstr$UNIX2003"
    )]
    pub fn confstr(name: ::c_int, buf: *mut ::c_char, len: ::size_t) -> ::size_t;
}
```

No need to use `cfg_if` if there is no `else` (I know we do this elsewhere, that will be a cleanup one day...)

Reply:
Nevermind, ignore this one.